src/scripts/jefferson

```diff
@@ -485,7 +585,7 @@ def main():
         if not os.path.exists(dest_path):
             os.mkdir(dest_path)
 
-        dump_fs(fs, dest_path)
+        dump_fs(fs, dest_path, verbose=args.verbose, force=args.force)
```

Comment:
Please move the addition of `verbose` mode into its own commit, same thing for `force` mode. We're trying to use atomic git commits.